VideoAPI/VideoApi.AcceptanceTests/VideoApi.AcceptanceTests.csproj

```diff
@@ -36,10 +38,9 @@
     <Content Include="Features\*.feature.cs" />
   </ItemGroup>
 
-  <ItemGroup>
-    <Compile Update="Features\Events.feature.cs">
-      <DependentUpon>Events.feature</DependentUpon>
-    </Compile>
-  </ItemGroup>
-
+    <ItemGroup>
```

Comment:
is this item group required? i  know it may exist but i don't  think it is necessary

Reply:
Removed :)